app/src/main/java/org/breezyweather/settings/preference/composables/ListPreference.kt

```diff
@@ -117,6 +122,7 @@ fun ListPreferenceView(
             elevation = if (enabled) defaultCardListItemElevation else 0.dp
         ) {
             ListItem(
+                colors = colors,
```

Comment:
I don't think it makes sense to add `colors` here as the background color of the card will be different from the one used for the list item. Currently, we don't have any issues with the card version of the preference composables so I'd leave it as-is for now.

The same applies to the [PreferenceView](https://github.com/breezy-weather/breezy-weather/pull/1293/files#diff-323f5c2cc2ba1e2d29d8c3e066af6601ab0a3575b232dd34cfe377f02192803dL91).

Removing that would also remove the need to pass `colors` in [WeatherSourcesSettingsScreen](app/src/main/java/org/breezyweather/settings/compose/WeatherSourcesSettingsScreen.kt).

Reply:
Makes sense to me. Changes in the latest commit.